admin/src/js/services/select2-search.js

```diff
@@ -26,18 +28,29 @@ angular.module('inboxServices').factory('Select2Search',
       return $(format.sender(row.doc, $translate));
     };
 
-    const defaultTemplateSelection = function(row) {
-      if (row.doc) {
-        return row.doc.name + (row.doc.muted ? ' (' + $translate.instant('contact.muted') + ')': '');
+    const defaultTemplateSelection = (selection) => {
```

Comment:
I'm noticing that when one of the facilities is deleted (because I deleted it as my user in the frontend - another but that needs to be addressed) - this is how the edit user UI looked: 
![image](https://github.com/medic/cht-core/assets/35681649/9d89de1e-f7da-4fca-98cf-f9dd039a9824)

The list of facilities had no names and there was an error in the console.

Reply:
I'm still seeing this behavior :(

Reply:
I think you need to check the results from your all docs request to see if any of the docs has been deleted.